website/docs/docs/build/documentation.md

```diff
@@ -60,13 +58,14 @@ models:
 
 </File>
 
-
 ## Generating project documentation
-You can generate a documentation site for your project (with or without descriptions) using the CLI.
+You can generate a documentation site for your project (with or without descriptions) in development using your editor or with [dbt Explorer](/docs/collaborate/explore-projects) in dbt Cloud after a successful job run in the deployment environment.
```

Comment:
@mirnawong1 i'd frame this as 'you can use explorer to view the latest metadata about your project, or generate a static docs site'. then point the reader to the explorer docs to read about setup (within the generate metadata section). no need to link to the docs job article for explorer setup. generating docs is still an option for folks who don't have explorer or want more flexibility, but shouldn't be our default suggestion.